### PR TITLE
Homepage - News You Can Use

### LINKED ISSUE #4940: [Frontend] News you can use
desktop

Screen Shot 2020-07-29 at 12 52 43 PM

tablet

Screen Shot 2020-07-29 at 12 52 57 PM

mobile

Screen Shot 2020-07-29 at 12 53 37 PM
Screen Shot 2020-07-29 at 12 53 42 PM

### PR DESCRIPTION
**Pull request by Pomax**
**(Will assign code reviewer once this passes design review)**

Eventually closes #4940 

Also note that this PR includes the factory work to make `load_fake_data` prepopulate the homepage with useful bindings.

### CODE REVIEW

**Comment by mmmavis** [comment]
To-do: 

Work with designers to address content overflow issue

image

**Comment by mmmavis** [comment]
@taisdesouzalessa This is ready for design review. I've made styling changes based on our convo in [this ticket](https://github.com/mozilla/foundation.mozilla.org/issues/4940). The featured image will have minimum height `400px` and the content box on the right will always be vertically centered and have minimum top/bottom margin 10px align.

**Comment by mmmavis** [comment]
@taisdesouzalessa I think and I hope I've addressed all issues you mentione above (including the overflow edge case that you found. Good catch!)

File: network-api/networkapi/wagtailpages/templates/wagtailpages/fragments/news_you_can_use.html [hunk 1]
```diff
@@ -1,41 +1,73 @@
...
 
-<div class="row" id="home-read-more-news">
-    <div class="col-12">
-      <h2 class="h4-heading">{% trans "News you can use" %}</h2>
-      <span>{% trans "Read more news →" %}</span>
-    </div>
+<div class="row section-news-you-can-use section-margin">
+  <div class="col-12 d-flex flex-row justify-content-between">
+    <h2 class="h4-heading mb-4">{% trans "News you can use" %}</h2>
+    {# this should not be a hardcoded route... #}
```
Comment (mmmavis) [inline network-api/networkapi/wagtailpages/templates/wagtailpages/fragments/news_you_can_use.html:6]: @Pomax do you remember why this should not be hardcoded?

File: source/sass/components/blog-authors.scss [hunk 2]
```diff
@@ -1,26 +1,32 @@
 .blog-authors {
-  .author-image {
```
Comment (mmmavis) [inline source/sass/components/blog-authors.scss:2]: wrapped in `.author-images {}` below

**Comment by mmmavis** [comment]
@taisdesouzalessa hopefully I don't have to bug you for another design review after this 😂 

I've updated the vertically padding for all `News you can use`, `Spotlight`, and `Quote` sections to 3rem/mobile and 5rem/tablet+.

File: network-api/networkapi/wagtailpages/templates/wagtailpages/fragments/news_you_can_use.html (hunk 1 above)
Comment (Pomax) [reply network-api/networkapi/wagtailpages/templates/wagtailpages/fragments/news_you_can_use.html:6]: becca renamed `/blog` to something else and that broke everthing that ties into blogs. So we should really figure out a way to "not hardcode this". I've filed https://github.com/mozilla/foundation.mozilla.org/issues/5085 so we can fix that later.

**Pull request merged**